aiohttp/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "3.9.4.dev0"
```

Comment:
Oh, does this mean that we forgot to bump it in the previous release? If did we bump it backwards? `3.9.4.dev0 < 3.9.4`

Reply:
Nope, that's just my mistake. Let me update the instructions...